src/cli_plugin/list/list.js

```diff
@@ -1,13 +1,28 @@
-import { statSync, readdirSync } from 'fs';
+import { statSync, readdirSync, readFileSync, accessSync, R_OK } from 'fs';
 import { join } from 'path';
 
+function readable(path) {
+  try {
+    accessSync(path, R_OK);
+    return true;
+  } catch (e) {
+    return false;
+  }
+}
+
 export default function list(settings, logger) {
   readdirSync(settings.pluginDir)
   .forEach((filename) => {
     const stat = statSync(join(settings.pluginDir, filename));
 
     if (stat.isDirectory() && filename[0] !== '.') {
-      logger.log(filename);
+      const packagePath = join(settings.pluginDir, filename, 'package.json');
+      if (readable(packagePath)) {
+        var packageJSON = JSON.parse(readFileSync(packagePath, 'utf8'));
```

Comment:
Since you only need `version`, I'd recommend using destructuring:

```
const { version } = JSON.parse(readFileSync(packagePath, 'utf8'));
```

One way or another, it should be a `const` rather than `var`.